pymod/test/test_chemistry/test_atom_list.py

```diff
@@ -1,8 +1,8 @@
 import os # noqa
 import sys # noqa
 sys.path.insert(0, '..') # noqa
-from generic import Coordinate # noqa
-from generic import AtomList # noqa
+from geometry import Coordinate # noqa
+from chemistry import AtomList, CellList # noqa
```

Comment:
I think CellList should be part of a more abstract "geometry" module.  It definitely does not belong in `chemistry`.

Reply:
Hmm okay. It seems to be `AtomList` with an extra method or two so Darren and I will have to talk about how to break this out of the chemistry module.